src/ui/public/notify/notify.less

```diff
@@ -77,6 +77,14 @@
     }
   }
 
+  .toast-banner {
+    // no flex
```

Comment:
What effect do we want this selector to have? Since it contains no properties, it's currently a no-op.